io/serialport/25-serial.js

```diff
@@ -483,6 +500,24 @@ module.exports = function(RED) {
                 else {
                     done();
                 }
+            },
+            connect: function(port) {
+                if (connections[port]) {
+                    try {
+                        connections[port].connect();
+                    }
+                    catch(err) { }
+                }
+            },
+            disconnect: function(port) {
+                if (connections[port]) {
+                    try {
+                        connections[port].disconnect(function() {
+                            RED.log.info(RED._("serial.errors.closed",{port:port}), {});
+                        });
+                    }
+                    catch(err) { }
```

Comment:
Do we really want to do nothing ? should this raise an error - or set node.status at least ?

Reply:
same here, try catch was not necessory as we are using existing function to disconnect so it should be handled in there.